src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -0,0 +1,222 @@
+            items_in_container.append(item)
+
+    return get_one_or_many(items_in_container)
+
+def show_container(client, container_name, vault, container_type="AzureVM", status="Registered"):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    backup_management_type = get_backup_management_type(container_type)
+
+    filter_string = get_filter_string({
+        'friendlyName' : container_name,
+        'backupManagementType' : backup_management_type,
+        'status' : status})
+
+    containers = containers_mgmt_client_factory(None).list(rs_vault.name, resource_group, filter_string)
+    return get_one_or_many(get_paged_list(containers))
+
+def list_containers(client, vault, container_type="AzureVM", status="Registered"):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    backup_management_type = get_backup_management_type(container_type)
+
+    filter_string = get_filter_string({
+        'backupManagementType' : backup_management_type, 
+        'status' : status})
+
+    containers = containers_mgmt_client_factory(None).list(rs_vault.name, resource_group, filter_string)
+    return get_one_or_many(get_paged_list(containers))
+
+def show_policy(client, policy_name, vault):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    policy = policy_mgmt_client_factory(None).get(rs_vault.name, resource_group, policy_name)
+    return policy
+
+def list_policies(client, vault):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    policies = policies_mgmt_client_factory(None).list(rs_vault.name, resource_group)
+    return get_one_or_many(get_paged_list(policies))
+
+def enable_protection_for_vm(client, vm, vault, policy):
+    policy_client = policy_mgmt_client_factory(None)
+
+    vm = get_vm_from_json(vm_mgmt_client_factory(None), vm)
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+    policy = get_policy_from_json(policy_client, policy)
+
+    vm_name = vm.name
+    vm_rg = get_resource_group_from_id(vm.id)
+    
+    protectable_item = get_protectable_item(rs_vault.name, resource_group, vm_name, vm_rg)
+
+    container_uri = get_container_uri_from_id(protectable_item.id)
+    item_uri = get_item_uri_from_id(protectable_item.id)
+    
+    vm_item_properties = AzureIaaSComputeVMProtectedItem(policy_id=policy.id, source_resource_id=protectable_item.properties.virtual_machine_id)
+    vm_item = ProtectedItemResource(properties=vm_item_properties)
+
+    resp = protected_items_mgmt_client_factory(None).create_or_update(rs_vault.name, resource_group, "Azure", container_uri, item_uri, vm_item)
+
+    return resp
+
+################# Private Methods
+def get_protectable_item(vault_name, vault_rg, vm_name, vm_rg):
```

Comment:
Python convention is private methods start with an underscore `_`.
Then you can remove the `Private Methods` comment.